src/App.tsx

```diff
@@ -1,93 +1,217 @@
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
-      </section>
-
-      <footer className="footer">
-        <span className="todo-count" data-cy="todosCounter">
-          3 items left
-        </span>
-
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">All</a>
-          </li>
-
-          <li>
-            <a href="#/active">Active</a>
-          </li>
-
-          <li>
-            <a href="#/completed">Completed</a>
-          </li>
-        </ul>
-
-        <button type="button" className="clear-completed">
-          Clear completed
-        </button>
-      </footer>
+      {todos.length > 0 && (
+        <>
+          <section className="main">
+            <input
+              type="checkbox"
+              id="toggle-all"
+              className="toggle-all"
+              data-cy="toggleAll"
+              checked={checked}
+              onChange={handleChangeCheckbox}
+            />
+            <label htmlFor="toggle-all">Mark all as complete</label>
+
+            <Routes>
+              <Route
+                path="/"
+                element={(
+                  <TodoList
+                    todos={todos}
+                    changeTodoStatus={changeTodoStatus}
+                    deleteTodo={deleteTodo}
+                    editTitle={editTitle}
+                  />
+                )}
+              />
+            </Routes>
+
+            <Routes>
+              <Route
+                path="active"
+                element={(
+                  <TodoList
+                    todos={activeTodos}
+                    changeTodoStatus={changeTodoStatus}
+                    deleteTodo={deleteTodo}
+                    editTitle={editTitle}
+                  />
+                )}
+              />
+            </Routes>
+
+            <Routes>
+              <Route
+                path="completed"
+                element={(
+                  <TodoList
+                    todos={completedTodos}
+                    changeTodoStatus={changeTodoStatus}
+                    deleteTodo={deleteTodo}
+                    editTitle={editTitle}
+                  />
+                )}
+              />
+            </Routes>
+          </section>
```

Comment:
Let's keep only Rotes here in the App. Other logic move to the TodoApp component:
<img width="821" alt="image" src="https://user-images.githubusercontent.com/52370890/183444376-70136505-e737-4373-806b-78be2dead18f.png">